cmd/storage.go

```diff
@@ -172,17 +172,57 @@ var storageListCmd = &cobra.Command{
 	},
 }
 
+var storageMountCmd = &cobra.Command{
+	Use:   "mount [storage name]",
+	Short: "mount storage to a component",
+	Example: `
+  # Mount storage 'dbstorage' to current component
```

Comment:
Please move this line above so it's in-line with `Example:` or else it get's an extra space within help.

```
	Example: `  # Mount storage 'dbstorage' to current component`
```

Like that ^^